source/beamline/CLS/CLSQE65000Detector.cpp

```diff
@@ -158,12 +158,18 @@ bool CLSQE65000Detector::initializeImplementation(){
 }
 
 bool CLSQE65000Detector::acquireImplementation(AMDetectorDefinitions::ReadMode readMode){
-	if(!isConnected() || readMode != AMDetectorDefinitions::SingleRead)
+	if(!isConnected() || readMode != AMDetectorDefinitions::SingleRead) {
+
+		if(readMode != AMDetectorDefinitions::SingleRead) {
```

Comment:
Nope, I added that conditional in order to put a debug in there to show the reason why this call was returning false. When I cleaned up the code ready for submission I got rid of all the debugs, but forgot to remove this conditional block.